rust/src/meas/gaussian/ffi.rs

```diff
@@ -4,29 +4,41 @@ use std::os::raw::{c_char, c_void};
 use num::Float;
 
 use crate::{err, try_, try_as_ref};
-use crate::core::{FfiResult, IntoAnyMeasurementFfiResultExt};
+use crate::core::{FfiResult, IntoAnyMeasurementFfiResultExt, SmoothedMaxDivergence, ZeroConcentratedDivergence};
 use crate::core::{AllDomain, VectorDomain};
 use crate::ffi::any::AnyMeasurement;
 use crate::ffi::util::Type;
-use crate::meas::{GaussianDomain, make_base_analytic_gaussian, make_base_gaussian};
+use crate::meas::{GaussianDomain, make_base_analytic_gaussian, make_base_gaussian, GaussianMeasure};
 use crate::traits::samplers::SampleGaussian;
-use crate::traits::{CheckNull, InfAdd, InfCast, InfLn, InfMul, InfSqrt, InfDiv, InfSub, InfExp};
+use crate::traits::{CheckNull, InfAdd, InfCast, InfLn, InfMul, InfSqrt, InfDiv, InfSub, InfExp, InfPow, ExactIntCast};
 
 #[no_mangle]
 pub extern "C" fn opendp_meas__make_base_gaussian(
     scale: *const c_void,
     D: *const c_char,
+    MO: *const c_char
 ) -> FfiResult<*mut AnyMeasurement> {
-    fn monomorphize<D>(scale: *const c_void) -> FfiResult<*mut AnyMeasurement> where
-        D: 'static + GaussianDomain,
-        D::Atom: 'static + Clone + SampleGaussian + Float + InfCast<f64> + InfSub + InfDiv + CheckNull + InfMul + InfAdd + InfLn + InfSqrt + InfExp {
-        let scale = *try_as_ref!(scale as *const D::Atom);
-        make_base_gaussian::<D>(scale).into_any()
-    }
+    fn monomorphize_1<T>(scale: *const c_void, D: Type, MO: Type) -> FfiResult<*mut AnyMeasurement> where 
```

Comment:
Super minor: Elsewhere we're using the form `monomorphize1` rather than `monomorphize_1`.